content/quality-council/meetings/2023-05-16-meeting.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 2023-05-16
+---
+
+# 2023-05-16
+
+## Presents
+
+* [@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS) : Member of the Quality Coucil
+* [@boubkerbribri](https://github.com/boubkerbribri) : Member of the Quality Coucil
+* [@AureRita](https://github.com/AureRita)
+* [@florine2623](https://github.com/florine2623)
+* [@Progi1984](https://github.com/Progi1984)
+
+## Report
+
+Like Pablo Borowicz said, each council can organize like it wants. So this is the goal of this meeting : define rules for the Quality Coucil.
+
+### Members of the council
+
+There are 3 profiles which are members of the Quality Coucil :
+* Issue Manager
+* Software Developer in Test
+* Software Tester
+
+Each member can be : 
+* either a maintainer (can participate and vote during the Quality Coucil),
+* or a committer (can participate during the Quality Coucil).
+
+Members are updated on the [organization project page]({{< relref "/project-organization/people-and-roles#quality-council" >}}) with their role.
+
+### Votes
+
+* Only maintainers can vote
+* Each maintainer has one vote
+* Results of the vote are public
+* Vote is accepted when a simple majority is reached
+
+### Lead Council
+
+* Actual Lead is [@Robin-Fischer-PS](https://github.com/Robin-Fischer-PS)
+* He is elected for one year (until 1st of April 2024) by maintainers.
```

Comment:
Leads are not elected by maintainers but chosen by Leadership council.

Reply:
I disagree with that : 

> For the time being, the project’s main sponsor, PrestaShop SA, reserves the right to appoint the Leadership Council’s lead.

Source : https://build.prestashop-project.org/news/2023/a-new-organization-for-the-open-source-project/